sdk/src/main/java/com/hcaptcha/sdk/HCaptcha.java

```diff
@@ -24,10 +24,11 @@ public final class HCaptcha extends Task<HCaptchaTokenResponse> implements IHCap
     private HCaptchaConfig config;
 
     @NonNull
-    private final IHCaptchaHtmlProvider htmlProvider = new HCaptchaHtml();
+    private HCaptchaSettings settings;
```

Comment:
`Settings` and `Config` are very similar, thus is hard to understand the distinction.

What do you think of naming this `private HCaptchaInternalConfig internalConfig`?

Reply:
Solved in https://github.com/hCaptcha/hcaptcha-android-sdk/pull/67/commits/5d391a106e0a8b913ce9bdecce6c3003a1b55963